README.md

```diff
@@ -386,3 +285,56 @@ python3 -m generative_data_prep pipeline --input_file_path=./tests/examples/meta
 ```
 
 > [View decoded output](tests/examples/metaICL/decoded_data_prepped_metaICL.txt)
+
+</br>
+
+## Understanding Command Outputs
+
+### Terminal Output
+The metrics associated with this dataset will be printed in the terminal. These metrics give some insight into how the data was packed into sequences, and information about the training dataset.
```

Comment:
We also include this in output_path/logs.log, maybe lets specify this? Maybe we can add an entire section specifying that you can check the log file in output_path to see details?

Reply:
I expanded the first sentence to specify that you can check `<OUTPUT DIR PATH>/logs.log` to see these details. Let me know if you think this works, or if I should break it off into a separate headered section.